config/routes.rb

```diff
@@ -20,7 +20,7 @@
 
   resources :items, only: [:new, :show]
   resources :users, only: [:show, :create]
-  # resources :admins
-
+  resources :orders, only: [:new, :create, :show]
```

Comment:
@carmer @s-espinosa, would there be benefit to nesting order under user? technically the session[:cart] belongs to a user... not sure how to ask this.